dev-docs/source/Testing/SliceViewer/SliceViewer.rst

```diff
@@ -0,0 +1,168 @@
+
+- Create an Integrated PeaksWorkspace:
+
+.. code-block:: python
+
+	peaks = mtd['peaks']
+	integrated_peaks = IntegratePeaksMD(InputWorkspace='md_non_ortho', PeaksWorkspace='peaks',\
+	     PeakRadius=0.12, BackgroundOuterRadius=0.2, BackgroundInnerRadius=0.16)
+
+
+Tests
+-----
+
+Remember to SliceView 2D,3D,4D and non-orthogonal data. To enable non-orthogonal axes view, select the tri-fold toolbar button.
+
+.. figure:: ../../../../docs/source/images/wb-sliceviewer51-nonorthobutton.png
+   :class: screenshot
+   :align: center
+
+
+1. Viewing Data
+###############
+
+For the data types above:
+	- Change the number of bins displayed
+	- Move the sliders
+	- Edit color limits, colormap, scale(lin/log), etc.
+
+2. Select Axes
+##############
+
+- Change the axes that are displayed by selecting the relevant ``X`` and ``Y`` axes in the top left. This is more interesting for higher dimension data.
+
+3. Cursor Tracking
+##################
+
+- Toggle "Track Cursor" on/off and check the cursor data makes sense
+- For a MatrixWorkspace there is much more information than for an MDWorkspace. See :ref:`Cursor Information Widget<mantid:sliceviewer_cursor>` for more details.
+
+4. Peak Overlay
+###############
+
+.. figure:: ../../../../docs/source/images/wb-sliceviewer51-peaksbutton.png
+   :class: screenshot
+   :align: center
+
+- Select the peak overlay button to choose which PeakWorkspace/s to overlay
+- Click on different peaks in the peak sorting table to display them
+- Try overlaying multiple peaks workspaces
+- Overlay Integrated peaks and observe the peak radius and background shell 
+  (see *Calculations* section of :ref:`algm-IntegratePeaksMD`) as displayed in the image below.
+- Click on a column title in the peak table to sort by that value, such as ``DetID``
+- Zooming in on peaks, and check that the data and peak move together 
+- Repeat step 2. (Select Axes) with peaks shown 
+- Peak overlay should not be shown for 2D data
+
+.. figure:: ../../../../docs/source/images/wb-sliceviewer51-peaksoverlay.png
+   :class: screenshot
+   :width: 75%
+   :align: center
+
+5. Toolbar buttons for changing axis limits
+###########################################
+
+- Home 
+- Pan + Stretch
+- Zoom (dynamic rebinning to ``_svrebinned`` workspace) - both by selecting region with mouse and scrolling
+
+6. Toggle gridlines on/off
+##########################
+
+- For normal and non-orthogonal axes data
+
+7. Line subplots and Region of Interest integration
+###################################################
+
+**(disabled for non-orthogonal data)**
+
+.. figure:: ../../../../docs/source/images/wb-sliceviewer51-roibutton.png
+   :class: screenshot
+   :align: center
+
+- Select the toolbar button for region of interest integration
+- Draw a shape to integrate over on the image and notice the line subplots change
+- Test the keyboard options in the bottom status bar message to output roi/cuts to workspaces
```

Comment:
I think keyboard shortcuts are very useful. It might be worth it to spell them out here. x and y are for profiles, c will produce both x and y, r will slice with thickness (rectangular region of interest). x, y, c will work both with thickness and for line profiles.